### DISCUSSION TITLE
No se que es jekyll

### DISCUSSION THREAD

**Discussion by Silvina-web** [community/community#34605] [Other Feature Feedback, Questions, & Ideas]
Hice un sitio a modo de practica para aprender js pero cuando oy al github pages la parte del js no la muestra y me sale este error

dice build with jekyll. No se que es jekyll.

Que tengo que hacer?
![image](https://user-images.githubusercontent.com/72451639/193069411-a75eff36-5099-4251-a4dd-896fea9182fc.png)

**Comment by radar-blip** [comment]
[GitHub Pages and Jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll)

It looks like you have fixed your site. Good work!
